package.json

```diff
@@ -6,7 +6,7 @@
   "module": "es/index.js",
   "jsnext:main": "es/index.js",
   "scripts": {
-    "test": "node_modules/.bin/jest",
+    "test": "cross-env NODE_ENV=commonjs node_modules/.bin/jest",
```

Comment:
I don't think everyone has cross-env installed! Could you add this to dev dependencies?